docs/user-guide/quick-start/contributor-gui.md

```diff
@@ -167,12 +240,18 @@ data types like libraries and preparations.
 
 Data can be linked to Library or Preparation metadata by using **Library ID** and **Preparation ID**.
 
+<figure markdown="span">
 ![Link Data](quick-start-images/linking-data.png)
+<figcaption>Select an experimental data file. The data must include a column to be linked to the sample metadata file (typically the <strong>SampleSource ID</strong>)</figcaption>
```

Comment:
Could you please rename here and a bit above in the text "SampleSource" to "Sample Source"? (I've just noticed that in the ODM Default Template this attribute is called "Sample Source ID", but in the text we mention this attribute as "SampleSource ID" or "SampleSourceID" and also "Sample Source ID" attribute is kinda hardcoded in some places. Since you also use this attribute name in some captures, could you please update it to the proper one?